platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/iss/internal/ConsensusHashFinder.java

```diff
@@ -205,28 +205,28 @@ public Map<Hash, HashPartition> getPartitionMap() {
     }
 
     /**
-     * Get the total stake in the network.
+     * Get the total weight in the network.
      */
-    public long getTotalStake() {
-        return totalStake;
+    public long getTotalWeight() {
+        return totalWeight;
     }
 
     /**
-     * Get the amount of stake of nodes that have submitted a hash for this round.
+     * Get the amount of weight of nodes that have submitted a hash for this round.
      */
-    public long getHashReportedStake() {
-        return hashReportedStake;
+    public long getHashReportedWeight() {
+        return hashReportedWeight;
     }
 
     /**
      * Write information about hash partitions to a string builder.
      */
     public void writePartitionData(final StringBuilder sb) {
         sb.append("Nodes holding ")
-                .append(hashReportedStake)
+                .append(hashReportedWeight)
                 .append("/")
-                .append(totalStake)
-                .append(" stake have reported a hash for round ")
+                .append(totalWeight)
+                .append(" weight have reported a hash for round ")
```

Comment:
A bit awkward wording, would read better as `weight of nodes have reported a hash for round`

Reply:
the phrases is embedded in a larger string builder.  This is the total statement: 
``` 
Nodes holding (# partial)/(# total) weight have reported a hash for round (# round).
```
vs 
```
Nodes holding (# partial)/(# total) weight of nodes have reported a hash for round (# round).
```
I am deciding to go with the following: 
``` 
Nodes holding (#)/(#) of total weight have reported a hash for round (#).
```